packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -84,13 +84,20 @@ class HotRunner extends ResidentRunner {
     return true;
   }
 
+  Future _reloadSourcesService(String isolateId,
```

Comment:
`Future<Null>` is probably better, I'll also add an exception to allow the service to return a failure message